modules/store/src/index.ts

```diff
@@ -14,6 +14,7 @@ export {
 export { createAction, props, union } from './action_creator';
 export { Store, select } from './store';
 export { combineReducers, compose, createReducerFactory } from './utils';
+export { setNgrxMockEnvironment, isNgrxMockEnvironment } from './flags';
```

Comment:
I suppose not, but is there a way to make the `setNgrxMockEnvironment` private?
This, to prevent the usage of it to disable the warning.